internal/fluentbit/config/builder/config_builder.go

```diff
@@ -10,7 +10,9 @@ import (
 )
 
 var (
-	ErrUndefinedOutputPlugin = errors.New("output plugin not defined")
+	ErrUndefinedOutputPlugin     = errors.New("output plugin not defined")
+	ErrInvalidPipelineDefinition = errors.New("invalid pipeline definition")
```

Comment:
Can we reuse `InvalidPipelineDefinition` for output and input problems?

Reply:
done